foreshadow/utils/common.py

```diff
@@ -106,3 +107,27 @@ def configure_column_sharer(self, column_sharer):
         """
         if hasattr(self, "column_sharer"):
             self.column_sharer = column_sharer
+
+
+class CustomizeParamsMixin:
+    """Mixin that customizes params returned from get_params."""
+
+    def customize_serialization_params(self, params):
+        """Select params that are only necessary.
```

Comment:
Update this doc string. I believe it should say: "Select subset of get_params on this class' init."
Two major questions:
1. (for my own knowledge). Why was this needed instead of just using get_params?
2. Shouldn't "self" and "kwargs" already not be in the params since they were returned from self.get_params()?
In which case, see my comment below about the for loop and how it could be augmented.